### ISSUE TITLE
Beacon interferes with ediff

### ISSUE THREAD

**Issue by Knusper** [Malabarba/beacon#56]
Beacon mode causes errors when `ediff` is run.  Probably everything `ediff` related should be put on the `beacon-dont-blink-major-modes` list globally.

**Comment by Malabarba** [comment]
Makes sense, could you provide a list of ediff major-modes? I can add them to the list then.
